src/cli/progress_bar.rs

```diff
@@ -32,6 +32,15 @@ impl ProgressBar {
         println!("{}", &self.end_message);
     }
 
+    pub fn progress_unknown(&self) {
+        self.pb.set_style(
+            ProgressStyle::default_spinner()
+                .tick_strings(cli::spinner::TICKS)
+                .template("{spinner:.blue} {msg} {bytes}")
```

Comment:
I would remove the `{bytes}` from the template and just display the spinner

Reply:
Hm, I wanted this to be similar to how `xh` handles these downloads, and personally I prefer seeing at least some number go up 😅

Reply:
Ok let's leave it as feedback to the user